package.json

```diff
@@ -117,8 +116,9 @@
     "start-server-and-test": "^2.0.8",
     "storybook": "^7.0.2",
     "style-loader": "^0.23.1",
-    "styled-components": "^5.3.9",
+    "styled-components": "^6.1.15",
```

Comment:
If a customer has v5 styled component peer dependency install, would this work?

Reply:
Yes, the demo and react component library will both use v5 styled components. I've verified storybook, meeting demo with v5 styled components.